README.md

```diff
@@ -51,33 +52,39 @@ Just paste the following URL in your profile readme and you are good to go.
 
 ### [Attention ⚠](#Deploy-on-your-own-heroku-instance)
 
-## Use themes
+---
+
+## Use Themes
 
 _`username=ashutosh00710&theme=theme_name`_
 
 ```md
 [![Ashutosh's github activity graph](https://github-readme-activity-graph.cyclic.app/graph?username=Ashutosh00710&theme=dracula)](https://github.com/ashutosh00710/github-readme-activity-graph)
 ```
 
-[Manual Customization](#customization) is also available
+[Manual Customization](#customization) is also available.
+
+---
 
 ## Available Themes
 
 |            Name            |                            Preview                            |
 | :------------------------: | :-----------------------------------------------------------: |
 | **Default (cotton candy)** |    <img src="./asset/default.svg" height=250 alt="graph"/>    |
-|         **react**          |     <img src="./asset/react.png" height=250 alt="graph"/>     |
-|       **react-dark**       |  <img src="./asset/react-dark.svg" height=250 alt="graph"/>   |
-|         **github**         |    <img src="./asset/github.svg" height=250 alt="graph"/>     |
-|     **github-compact**     | <img src="./asset/github-compact.svg" height=250 alt="graph"/>|
-|         **xcode**          |     <img src="./asset/xcode.svg" height=250 alt="graph"/>     |
-|         **rogue**          |     <img src="./asset/rogue.svg" height=250 alt="graph"/>     |
-|         **merko**          |     <img src="./asset/merko.png" height=250 alt="graph"/>     |
-|          **vue**           |      <img src="./asset/vue.png" height=250 alt="graph"/>      |
-|      **tokyo-night**       |  <img src="./asset/tokyo-night.png" height=250 alt="graph"/>  |
-|     **high-contrast**      | <img src="./asset/high-contrast.png" height=250 alt="graph"/> |
-
-For more themes click [here](https://github.com/Ashutosh00710/github-readme-activity-graph/blob/main/THEMES.md)
+|         **React**          |     <img src="./asset/react.png" height=250 alt="graph"/>     |
```

Comment:
Makes more sense to have the theme names lower case since that's the way they're added to the url

Reply:
Changes were made as requested for all given theme names.